### ISSUE TITLE
Format Exception in browser

### ISSUE THREAD

**Issue by guyluz11** [aissat/easy_localization#316]
I am getting this from some of my users in chrome version `87.0.4280.77` and `87.0.4280.141` on laptop and mobile

![Exception](https://user-images.githubusercontent.com/9304740/104192072-01778580-5427-11eb-864b-a10e381f18ed.jpeg)

pub version:
 `easy_localization: ^2.3.3`
 
flutter version:
 `flutter_macos_1.26.0-1.0.pre-dev`

It is working for me, if you want you can test the site [https://cbjinni.com](https://cbjinni.com)

**Comment by Overman775** [comment]
@guyluz11  https://cbjinni.com works fine, I don't see an error

**Comment by guyluz11** [comment]
I don't see errors too, but some of the users that used my site did.
If you need more information for this please tell me what do you need.

**Comment by Overman775** [comment]
@guyluz11 json with locale keys can't be loaded and this request redirect to index page, check files name

maybe try use `CodegenLoader()`

**Comment by guyluz11** [comment]
I found the bug.

![image](https://user-images.githubusercontent.com/9304740/106269633-6f98b680-6235-11eb-9797-3dcc2980eef3.png)

I have add `Local ('he', 'IL')` when I didn't add the file to `translations` folder, so that bag is effecting only people that have `he IL` as their OS/browser language.
Tested on my own phone.

I will close the issue now, if you think that it is something that need to be solved and be fix from this packages than you are free to reopen this issue.
